rust/agama-lib/src/software/model/registration.rs

```diff
@@ -41,6 +41,23 @@ pub struct AddonParams {
     pub registration_code: Option<String>,
 }
 
+/// Addon registration
+#[derive(Clone, Serialize, Deserialize, utoipa::ToSchema)]
+#[serde(rename_all = "camelCase")]
+pub struct AddonProperties {
+    // Addon identifier
```

Comment:
it is part of schema, so `///` should be used to see documentation in schema...also all properties should have user documentation.